### PR TITLE
pr split five - add five files

### PR DESCRIPTION
**Pull request by michaelkernaghan**
Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: integration-tests/contract-estimation-tests.spec.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ CONFIGS().forEach(({ lib, setup, knownBaker, createAddress, protocol, rpc }) =>
   const jakartanet = (protocol === Protocols.PtJakart2) ? test : test.skip;
   const kathmandunet = (protocol === Protocols.PtKathman) ? test : test.skip;
 
-  describe(`Estimate scenario using: ${rpc}`, () => {
+  describe(`Test estimate scenarios through contract api using: ${rpc}`, () => {
```
Comment (roxaneletourneau) [inline integration-tests/contract-estimation-tests.spec.ts:13]: We are testing the `estimate` not `through contract api`

File: integration-tests/contract-failwith.spec.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
       }
 
       try {
-        // Bypass estimation by specifyint fee & limits
+        // Bypass estimation by specify int fee & limits
```
Comment (roxaneletourneau) [inline integration-tests/contract-failwith.spec.ts:30]: `specify int` -> specifying

File: integration-tests/contract-lambda-view.spec.ts [hunk 3]
```diff
@@ -8,13 +8,15 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
   const toJSON = (x: any) => JSON.parse(JSON.stringify(x));
 
-  describe(`Lambda view using: ${rpc}`, () => {
+  const test = require('jest-retries');
```
Comment (roxaneletourneau) [inline integration-tests/contract-lambda-view.spec.ts:11]: Remove jest-retries?

**Review by roxaneletourneau — CHANGES_REQUESTED** [review]

File: integration-tests/contract-estimation-tests.spec.ts (hunk 1 above)
Comment (michaelkernaghan) [reply integration-tests/contract-estimation-tests.spec.ts:13]: changed

File: integration-tests/contract-failwith.spec.ts (hunk 2 above)
Comment (michaelkernaghan) [reply integration-tests/contract-failwith.spec.ts:30]: fixed

File: integration-tests/contract-lambda-view.spec.ts (hunk 3 above)
Comment (michaelkernaghan) [reply integration-tests/contract-lambda-view.spec.ts:11]: removed

**Review by roxaneletourneau — APPROVED** [review]

**Pull request merged**